.circleci/config.yml

```diff
@@ -265,12 +288,25 @@ jobs:
           path: vendor/dep.dot
       # TODO: auto-commit Gopkg.lock if test is successful
 
+  # Runs first, will download any dependencies - currently envoy, and do any work
+  # that can be shared between build/test/lint.
   dependencies:
     <<: *defaults
     steps:
       - checkout
       - run: make submodule-sync
       - run: make init
+      - run: git.pullmaster
```

Comment:
you have it above the submodule-sync in the other build above; it probably should be first in case there is a vendor change in master